packages/components/src/internal/components/user/actions.ts

```diff
@@ -74,6 +74,15 @@ export function getUserRoleDisplay(user: User): string {
     }
 
     if (hasAllPermissions(user, [PermissionTypes.Read])) {
+        if (
```

Comment:
minor: it is find to keep this if statement nested within the reader check, but I think it would be moved out above it to be more like the other checks because if the user didn't have read, they probably aren't going to see the message about their role display anyway.